x-pack/plugins/event_log/generated/schemas.ts

```diff
@@ -116,6 +116,25 @@ export const EventSchema = schema.maybe(
             action_subgroup: ecsString(),
             status: ecsString(),
             outcome: ecsString(),
+            summary: schema.maybe(
+              schema.object({
+                new: schema.maybe(
```

Comment:
Should this be `schema.maybe`? As long as it's a summary action we will always  have  counts either 0 or greater.

Reply:
These changes were auto generated, so I am not sure I can change them

Reply:
you are right, it looks like our script puts everything in `schema.maybe()`